src/getSurfSubModNames.c

```diff
@@ -1918,6 +1918,195 @@ global_model_parameters *getGlobalModelParameters(char *modelVersion, char *TOPO
         GLOBAL_MODEL_PARAMETERS->basinBoundaryNumber[basinNum][1] = 0;
         
         
+        GLOBAL_MODEL_PARAMETERS->nBasins = 9;
```

Comment:
Just making sure we wanted to make changes to 2.0.1 too?

Otherwise looks good.

Reply:
thats the original line. if you scroll down to the bottom, thats where the original line was.
So for some reason git marked the original line as changed but added/modified line as unchanged.